models/cataract.py

```diff
@@ -26,10 +29,26 @@ def predict(age: str, gender: str) -> PredictCataractResponseSchema:
     # model.predict
     # return results
 
+    # loading the model and weights
+    model = model_from_json(open("o_model.json").read())
+    model.load_weights("model_o_weights.h5")
+
+    # process input image
+    image = cv2.imread(absolute_image_path, cv2.IMREAD_COLOR)
+    image = cv2.resize(image, (200, 200))
+    image = np.array(image).reshape(-1, 200, 200, 3)
+
+    # predict results
+    pred = model.predict(image)
+    if pred > 0.5:
+        outcome = "Cataract"

```

Comment:
NIT - cc @JLZJ - maybe we can consider changing to "Absent" / "Present" to maintain consistency with the coronary, diabetes, and stroke models?